simTest/EntriesBeforeChain_test.go

```diff
@@ -29,89 +26,60 @@ func TestCreatEntriesBeforeChain(t *testing.T) {
 
 	numEntries := 250 // set the total number of entries to add
 
-	t.Run("generate accounts", func(t *testing.T) {
-		println(b.String())
-		println(a.String())
-	})
+	println(b.String()) // print factoid & ec addresses
+	println(a.String())
 
-	// KLUDGE: using "LAF" causes timeout on CI
-	t.Run("Run sim to create entries", func(t *testing.T) {
-		state0 := SetupSim("L", map[string]string{"--debuglog": ""}, 200, 0, 0, t)
+	params := map[string]string{"--debuglog": ""}
+	state0 := SetupSim("L", params, 8, 0, 0, t)
```

Comment:
Probably should have LF so this is testes as both leader and follower

Reply:
set to `LLAAFF` in parchment